src/View/ViewBuilder.php

```diff
@@ -322,7 +322,7 @@ public function build($vars = [], Request $request = null, Response $response =
             $className = App::className($className, 'View', 'View');
         }
         if (!$className) {
-            throw new MissingViewException([$this->_className]);
+            throw new MissingViewException(['class' => $this->_className]);
```

Comment:
@ADmad I'm not following, MissingTemplateException already exist but in this case it's actually a missing view class I believe, right ?